integration/spark/src/main/scala/org/apache/spark/sql/hive/CarbonStrategies.scala

```diff
@@ -17,8 +17,6 @@
 
```

Comment:
It seems there is a problem in rebasing. It hasn't taken latest commits. This class not supposed to change.You overriding with old commit.